docker-compose.yml

```diff
@@ -249,6 +249,15 @@ services:
     ports:
       - "127.0.0.1:16686:16686"
 
+  prometheus:
```

Comment:
You might want to add a profile just like jaeger to make this service optional.

Maybe even a good idea to have a shared profile with jaeger; such as 'monitoring' (or 'development')